js/widgets/panel.js

```diff
@@ -360,29 +361,34 @@ return $.widget( "mobile.panel", {
 					}
 
 					if ( o.theme && o.display !== "overlay" ) {
-						self._page().parent()
-							.addClass( o.classes.pageContainer + "-themed " + o.classes.pageContainer + "-" + o.theme );
+						self._addClass( self._page().parent(),
+							classesMap.pageContainer + "-themed " +
+							classesMap.pageContainer + "-" + o.theme );
```

Comment:
I added to the increase code maintainability since the same string was being used all over the place. Replacing the value of the variable replaces the value at every place where the variable is used.